### PR TITLE
Sos 453 agent for algernon

### PR DESCRIPTION
**Pull request by aylusltd**
It's not there yet but putting this on your radar.

### CODE REVIEW

**Comment by aylusltd** [comment]
Finished

File: verinfast/cloud/aws.py [hunk 1]
```diff
@@ -5,7 +5,6 @@
 from utils.utils import debugLog
 
 def runAws (targeted_account, start, end, path_to_output):
-
```
Comment (SeanTConrad) [inline verinfast/cloud/aws.py:8]: This file needs to be moved on my machine, or it breaks the import of things in the aws path.

File: verinfast/agent.py [hunk 2]
```diff
@@ -53,6 +53,10 @@
 from utils.utils import debugLog
 from cloud.aws import runAws
```
Comment (SeanTConrad) [inline verinfast/agent.py:54]: I would change to `from cloud.aws.aws` import runAws

File: verinfast/cloud/aws/instances.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
+import json
+import os
+
+import boto3
+
+import regions as r
```
Comment (SeanTConrad) [inline verinfast/cloud/aws/instances.py:6]: On my machine, this needs to be:
`import cloud.aws.regions as r`
Only the parent script that is run has its path in the sys path.

File: verinfast/cloud/gcp/instances.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
+import json
+import os
+
+from google.cloud import compute_v1
+
+from zones import zones
```
Comment (SeanTConrad) [inline verinfast/cloud/gcp/instances.py:6]: Probably needs to be:
`import cloud.gcp.zones as zones`

**Review by SeanTConrad — CHANGES_REQUESTED** [review]

File: verinfast/agent.py [hunk 5]
```diff
@@ -408,18 +412,28 @@ def scanCloud(config):
...
             checkDependency("aws", "AWS Command-line tool")
 
-            aws_file = runAws(targeted_account=provider["account"], start=provider["start"], end=provider["end"], path_to_output=output_dir)
-            debugLog(msg=aws_file, tag="AWS Results")
-            upload(file=aws_file, route=f"/report/{config['report']['id']}/Costs", source="AWS")
-        
+            aws_cost_file = runAws(targeted_account=provider["account"], start=provider["start"], end=provider["end"], path_to_output=output_dir)
+            debugLog(msg=aws_cost_file, tag="AWS Costs")
+            upload(file=aws_cost_file, route=f"/report/{config['report']['id']}/Costs", source="AWS")
+            aws_instance_file = get_aws_instances(accountId=provider["acconut"], path_to_output=output_dir)
```
Comment (SeanTConrad) [inline verinfast/agent.py:418]: "acconut" should be "account"

**Review by SeanTConrad — APPROVED** [review]

**Pull request merged**